python/cuml/cuml/tests/test_linear_model.py

```diff
@@ -693,7 +693,7 @@ def test_logistic_regression_model_digits(
     # smallest sklearn score with max_iter = 10000
     # put it as a constant here, because sklearn 0.23.1 needs a lot of iters
     # to converge and has a bug returning an unrelated error if not converged.
-    acceptable_score = 0.95
+    acceptable_score = 0.94
```

Comment:
How about lowering this even more? Mostly to avoid us coming here again in the future to lower it to `0.93`, and then a few months later again, etc.

Maybe something like `0.90` is good enough to detect that "things broadly work" and would make it very unlikely that we get unlucky and see it fluctuate down. I assume this test is more of a smoke test/check that things generally work. Not one that checks that there are no bugs in the algorithm's implementation.